editor/src/clj/editor/atlas.clj

```diff
@@ -473,15 +486,66 @@
 (defn- call-generator [generator]
   ((:f generator) (:args generator)))
 
-(defn- generate-packed-image [{:keys [_node-id image-resources texture-set-data-generator]}]
+(defn- generate-packed-image [{:keys [_node-id image-resources layout-data-generator]}]
```

Comment:
We don't need the fully compiled `TextureSetResult` in order to generate the packed image - only the layout-related parts of it.